akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -25,6 +25,12 @@ to route the message with the entity id to the final destination.
 Cluster sharding will not be active on members with status :ref:`WeaklyUp <weakly_up_java>` 
 if that feature is enabled.
 
+.. warning::
+   **Don't use Cluster Sharding together with Automatic Downing**,
```

Comment:
This should explain though that we specifically mean auto-down,
now it sounds like any automatic one is doomed.

I would not mind a link to SBR here too